CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+# Contributing In General
+So, you've come here hoping to make Concourse better?  Great!
+Concourse welcomes pull-request on any one of the many open-source
+pieces we currently have within our
+[organization](https://github.com/concourse).
+
+There are only a few ground rules that we like to see respected for
+your pull-request to be considered:
+
+- Make sure that you submit all pull-requests to the develop branch on
+Concourse (they won't build otherwise).
+- Each corresponding change should have an associated test to go along
+with it.  If you are having trouble testing, open an issue to discuss
+it.
+- Please don't forget to update the Concourse 
+[documentation](https://github.com/concourse/concourse/tree/develop/docs) 
+if you make a change to any the behavior (especially in fly).
+- Double check that all of the tests you have written pass on the
+pull-request
+
+With those ground rules out of the way, let's get you setup to work on
+the project!
+
+# Setup
+
+### Tools you will need
+- ruby
+    - use system provided if you have it
+- bosh_cli (ruby-gem and all around bosh goodness)
+    - `gem install bosh_cli bosh_cli_plugin_micro --no-ri --no-rdoc`
+- direnv (homebrew installed)
+     - `brew install direnv`
+- fly (grab this from Concourse.ci)
+    - [fly-binary-darwin](https://ci.concourse.ci/api/v1/cli?arch=amd64&platform=darwin)
+    - [fly-binary-linux](https://ci.concourse.ci/api/v1/cli?arch=amd64&platform=linux)
+- go (you can install via homebrew)
+    - `brew install go`
+- ginkgo (testing framework for go, assuming you grab go first)
+    - `go get github.com/onsi/ginkgo/ginkgo`
+- postgresql (you can also install this via homebrew)
+    - `brew install postgresql`
+- virtualbox
+    - `brew install virtualbox`
+- vagrant
+    - `brew install vagrant`
+
+### Setting Up a Bosh-lite
+Concourse is a bosh release, so you're probably going to want to setup a 
+bosh-lite that you can deploy concourse to before pushing your changes
+to the develop branch.
+
+Jump over to the bosh-lite [repo](https://github.com/cloudfoundry/bosh-lite) 
+and follow the instructions provided 
+[for virtual-box](https://github.com/cloudfoundry/bosh-lite)
```

Comment:
Guessing this was supposed to link here:

https://github.com/cloudfoundry/bosh-lite#using-the-virtualbox-provider

Also I think it's VirtualBox or virtualbox and never hyphenated.